### PR TITLE
feat: spec kakarot

### PR DESCRIPTION
**Pull request by BabyScope**
./scripts/test_spec_full.sh cookbook/specs/spec_add_kakarot.json jsonrpc https://sepolia-rpc.kakarot.org

Average block sync time is now where to be found. I let the script run for 2 hours and get the average block time with the results i got. Asked the dev team also but no response of them.

## Summary by CodeRabbit

- **New Features**
  - Introduced support for relaying data on Kakarot with configuration options including block distance, block time, API settings, and reliability thresholds.

### CODE REVIEW

File: cookbook/specs/kakarot.json [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+                    {
+                      "expected_value": "0x6b6b7274"
+                    }
+                  ]
+                },
+                {
+                  "name": "pruning",
+                  "values": [
+                    {
+                      "latest_distance": 847
```
Comment (Yaroms) [inline cookbook/specs/kakarot.json:48]: pruning should be for 24H of blocks
24*60*60/average_block_time = 8640

File: cookbook/specs/kakarot.json [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+                    }
+                  ]
+                }
+              ],
+              "extensions": [
+                {
+                  "name": "archive",
+                  "cu_multiplier": 5,
+                  "rule": {
+                    "block": 838
```
Comment (Yaroms) [inline cookbook/specs/kakarot.json:62]: same here

File: cookbook/specs/kakarot.json (hunk 1 above)
Comment (BabyScope) [reply cookbook/specs/kakarot.json]: Resolved in fix

File: cookbook/specs/kakarot.json (hunk 2 above)
Comment (BabyScope) [reply cookbook/specs/kakarot.json]: resolved in fix

File: cookbook/specs/kakarot.json (hunk 1 above)
Comment (BabyScope) [reply cookbook/specs/kakarot.json]: https://github.com/lavanet/lava/pull/1447/commits/86e3706b61d94ad14dd755e8fd2445de9808564b

see here

**Review by Yaroms — APPROVED** [review]

**Review by omerlavanet — APPROVED** [review]

**Pull request merged**